SMLHelper/FishFramework/CustomFish.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using UnityEngine;
+using SMLHelper.V2.Handlers;
+
+namespace SMLHelper.V2.FishFramework
+{
+    public class CustomFish
```

Comment:
This could become FishHandler

Reply:
I kept this class, and created a new class which will basically do what the Register() function does but externally, in keeping with SMLHelper's style

Reply:
I agree with this approach.